.github/workflows/build.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     - name: setup go
       uses: actions/setup-go@v2
       with:
-        go-version: '1.18'
```

Comment:
i'm not really sure what's going on here - this particular change should not be in the diff in my opinion - it's already set to 1.20 in main and on this branch.

Reply:
I think you need to merge _again_ `ooni/main` into the working branch after https://github.com/ooni/minivpn/commit/d976c5ab4eb66f4f3b3b811ae0ce325a2adaa809 has been merged; you're probably missing this commit in your working branch.